pages/linux/mmcli.md

```diff
@@ -0,0 +1,12 @@
+# mmcli
+
+> Control and monitor the ModemManager.
+> More information: <https://www.freedesktop.org/software/ModemManager/man/1.0.0/mmcli.8.html>.
+
+- List SMS messages available on the modem:
+
+`sudo mmcli -m {{modem}} --messaging-list-sms`
+
+- Delete a message from the modem by path:
+
+`sudo mmcli -m {{modem}} --messaging-delete-sms={{path}}`
```

Comment:
Based on our style guide we prefer `path/to/file`. Could this `path` be `path/to/message_file`? or is it a directory?
```suggestion
`sudo mmcli --modem={{modem}} --messaging-delete-sms={{path}}`
```

Reply:
I believe it can be, I think its files only so it's a fair suggestion.
Looking at the docs, I can see it accepts an index as well which I didn't realize, not 100% sure if I should document both or just leave it at one for simplicity.

For now, I'll leave it at path only since it's more explicit though, feel free to give your opinion!

Reply:
Even if it is a directory, we could still use `path/to/directory`. Or `path/to/file_or_directory` if both are possible.